### ISSUE TITLE
No "pyfiglet.fonts" module found

### ISSUE THREAD

**Issue by The1TrueJoe** [pwaller/pyfiglet#95]
I installed pyserial without the pip package manager. I could not get the SSL module to work on my Nvidia Jetson (ubuntu 14), so installing via pip was not an option. Instead, I cloned the repo and installed from the setup script.

This is not an issue when installing via pip on another computer (same python and pip versions).

Here are the commands I used:

```
git clone https://github.com/pwaller/pyfiglet
cd pyfiglet
sudo python3.7 setup.py install
```

When I checked out the most recent release tag, it solved the issue. (I deleted the previous version from site-packages)

```
git clone https://github.com/pwaller/pyfiglet
cd pyfiglet
git checkout d7079e9 # Working version
sudo python3.7 setup.py install
```

![Screen Shot 2022-05-02 at 1 38 33 AM](https://user-images.githubusercontent.com/47280152/166189284-5aa837f0-002e-4d74-8c89-a945356c2f06.png)

**Comment by Kraktorist** [comment]
The directory `pyfiglet/fonts` was removed for some reason. Maybe it's related to this issue #89 (Font licensing).
The one way to fix it is to create `pyfiglet/fonts` and put required fonts from `pyfiglet/fonts-standard` or/and `pyfiglet/fonts-contrib` to it.

**Comment by peterbrittain** [comment]
Yeah - I removed it so that we can build a package with all the fonts or just those that have a suitable license.  You need to pick which by running `make minimal` or `make full` first.  It's possible that there's a better way to specify that directly with setuptools, but I haven't looked as this project already used the Makefile.

**Comment by mkinney** [comment]
This should be fixed with https://github.com/pwaller/pyfiglet/pull/108

**Comment by pwaller** [comment]
@stefanor did you encounter this issue during #114 / #115, can anyone comment, does this remain an issue?

**Comment by peterbrittain** [comment]
The fix in #108 was definitely wrong.  The whole point of the split was to separate cleanly licensed fonts from ones with a less clear license (so that Linux distros can just pick the clean fonts).  

People using this codebase must use the makefile to decide which set they build.  Looks like 114 still respects that logic.

**Comment by pwaller** [comment]
Thanks for commenting @peterbrittain.

Judging from the error it looks like it lay within the pkg_resources logic which was replaced in #114 so I think it should no longer be an issue.

**Comment by pwaller** [comment]
Revising what I wrote last night: I take it the issue would still be present in some form, since it requires a 'make' invocation.. I guess it might be worth having some logic to load from the other directories if they're there, if someone would like to try and implement a fix.

**Comment by peterbrittain** [comment]
It depends what you want to achieve.

The distro builders will still want some way to ensure that they just pick up licensed fonts (while others possibly just want everything), so you still need some way to make two different packages.  With the legal requirements, you can't create one package with everything.  This isn't really what setup.py is designed to do.  

What's prompting the revisiting of this issue?

**Comment by pwaller** [comment]
It's that someone `git clone`'d the repository and finds that the software fails. In my mind, ideally it should work more or less as if you had installed it via a different route.

**Comment by peterbrittain** [comment]
So, the question is what do you want to build by default and how fo you propose to managevthe two different builds....

It would be triv8al to default builds to safevfonts by just renaming that directory to "fonts".

**Comment by peterbrittain** [comment]
Hopefully addressed by #122

**Comment by pwaller** [comment]
I'm going to close this as assumed fixed by #122, please ping this thread if you hit it again and I'll reopen, or file another issue.
